### ISSUE TITLE
Type installation for typescript

### ISSUE THREAD

**Issue by njfix6** [rollbar/rollbar-react#7]
How do we install the types for typescript for rollbar-react?

**Comment by johnpena** [comment]
Any updates on this? Has anyone figured this out?

**Comment by carvallegro** [comment]
Hey folks, I found a temporary workaround until this library becomes fully type compatible. I created a simple `rollbar-react.d.ts` file in my types folder on my project:

```ts
declare module '@rollbar/react'
```

Make sure that your TS config includes the types folder or files:
```diff
"include": [
-  "src/**/*"
+  "src/**/*",
+  "typing/rollbar-react.d.ts" // Or "typing/*"
]
```

This is very basic and won't provide your IDE with autocompletion since it doesn't declare anything but at least you will be able to use it.

**Comment by FriendlyUser** [comment]
Do you know how to adjust that logic to make sure at least `useRollbar()` is typed with Rollbar from `rollbar.js`?

**Comment by carvallegro** [comment]
The logic to declare **d.ts** files is in Typescript's official documentation:

- [Creating .d.ts Files from .js files](https://www.staging-typescript.org/docs/handbook/declaration-files/dts-from-js.html)
- [.d.ts Templates](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html)

**Comment by FriendlyUser** [comment]
I was looking for more of an example 

something that was leverage the temporary workaround in

```
declare module '@rollbar/react'
```

and tried

```
import Rollbar from "rollbar";

declare module '@rollbar/react' {
   function useRollbar(): Rollbar
}
```

And somehow has any number of functions defined for RollbarContext, etc...

Probably better to stick with the shortcut listed above and wait until the typings come out.

**Comment by quinnturner** [comment]
I've added a basic (and probably incorrect) set of TypeScript types to this project in #26.

**Comment by calvinf** [comment]
Fixed in #26 and #27

If you need help before next release, I put some info about how to inline into your project in comments on #26, but hopefully you can just pick up the next release.

**Comment by waltjones** [comment]
This is released in v0.10.0.
